bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -62,6 +62,19 @@ TEST_F(InstanceAdminIntegrationTest, ListInstancesTest) {
   }
 }
 
+/// @test Verify that InstanceAdmin::ListClusters works as expected.
+TEST_F(InstanceAdminIntegrationTest, ListClustersTest) {
+  // The emulator does not support cluster operations.
+  if (UsingCloudBigtableEmulator()) {
+    return;
+  }
+  auto clusters = instance_admin_->ListClusters();
+  for (auto const& i : clusters) {
+    auto const npos = std::string::npos;
+    EXPECT_NE(npos, i.name().find(instance_admin_->project_name()));
+  }
+}
```

Comment:
This test is trivial if there are no clusters in the project, not much you can do right now.  Please add a `TODO(#418) - create an instance and test that its cluster is returned here`.